### ISSUE TITLE
yolo.h5 file issue

### ISSUE THREAD

**Issue by DirenWitharana** [adnaneaabbar/object-detection-for-autonomous-driving#3]
Hi, there seems to be an issue with the yolo.h5 model file. Anyway to resolve it?

**Comment by adnaneaabbar** [comment]
You are right, here is a link on Kaggle to download the h5 file, since it's heavy I can't update the one in the repository, I will try to
[yolo.h5](https://www.kaggle.com/datasets/rmoharir8396/yolo-h5-file)

**Comment by DirenWitharana** [comment]
Thanks a lot for the prompt reply. I'll give this a try

**Comment by DirenWitharana** [comment]
Still getting an error 
![image](https://github.com/user-attachments/assets/0f117212-6d19-4d6b-b95b-45b8ac07888c)

**Comment by adnaneaabbar** [comment]
I think that your python version is too high for this project. This was 4 years ago, and if I remember correctly, there are other packages like cuDNN, tensorflow and keras that needed to be in a certain version also. I found  [something](https://stevejeffersonr.medium.com/fixing-model-compatibility-errors-across-tensorflow-versions-6a2114d85ba) that might help with your error.
